### PR TITLE
Fix: Time format issue for public event 

### LINKED ISSUE #5966: Public Event Sidebar Dates Scrumbled
There is an issue with the date/time of the event in the sidebar. It has an unidentifiable time and date.

Compare https://eventyay.com/e/d55cc647
Screenshot from 2020-12-09 01-52-37

### PR DESCRIPTION
**Pull request by nlok5923**
Fixes #5966 

#### Short description of what this resolves:
Time format changed from time-short to h:mm A to resolve the issue

#### Changes proposed in this pull request:

- format changed from time-short -> h:mm A

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/public/add-to-calender.hbs [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <i class="clock alternate outline icon pt-1"></i>
     <div class="content">
       {{#if this.isSingleDay}}
-        {{general-date @event.startsAt "dddd, D MMMM, YYYY" tz=@event.timezone}}<br>{{general-date @event.startsAt 'time-short' tz=@event.timezone}} {{t 'to'}} {{general-date @event.endsAt 'time-short' tz=@event.timezone}} 
+        {{general-date @event.startsAt "dddd, D MMMM, YYYY" tz=@event.timezone}}<br>{{general-date @event.startsAt  "h:mm A" tz=@event.timezone}} {{t 'to'}} {{general-date @event.endsAt "h:mm A" tz=@event.timezone}}
```
Comment (daretobedifferent18) [inline app/components/public/add-to-calender.hbs:6]: why

Comment (nlok5923) [reply app/components/public/add-to-calender.hbs:6]: As time-tz-short corresponds to h:mm A only

**Comment by nlok5923** [comment]
With time-tz-short it displays :
Screenshot (12)
with h:mm A
Screenshot (11)

**Comment by iamareebjamal** [comment]
Fixed in #5970

**Pull request closed without merging**